La'Tonia/acronym_create.py

```diff
@@ -0,0 +1,16 @@
+# accept user input of text/string
+# wrap input in str fn to ensure input translates as text/string
+user_input = str(input('Enter Phrase: '))
+
+# use split method to separate each character from user input
+text = user_input.split()
+
+# create empty string as assignment for variable for user input to cycle through before output
+char = ' '
+
+# create for loop to take each first character in the text/string from user input and change it into a capital letter
+for first_char in text:
```

Comment:
This is pretty nice - short a simple. This could be added to trinket for webpage?